src/legacy/core_plugins/data/public/index_patterns/index_patterns/index_patterns.ts

```diff
@@ -40,13 +41,16 @@ export class IndexPatterns {
   private savedObjectsClient: SavedObjectsClientContract;
   private savedObjectsCache?: Array<SimpleSavedObject<Record<string, any>>> | null;
   private apiClient: IndexPatternsApiClient;
+  private notifications: NotificationsSetup;
```

Comment:
@Avinar-24 @ppisljar Not sure that here `NotificationSetup` interface looks good.  Do we have something like `NotificationsContract` or `NotificationsBase`?

Reply:
NotificationSetup should be the setup contract of notifications service